cfy_manager/utils/certificates.py

```diff
@@ -581,3 +592,14 @@ def get_cert_and_key_filenames(new_cert_location,
 def get_ca_filename(new_ca_location, default_ca_location):
     return (new_ca_location if os.path.exists(new_ca_location)
             else default_ca_location)
+
+
+def _get_md5_hash(cert_path):
+    content = sudo(['openssl', 'x509', '-noout', '-modulus', '-in', cert_path])
+    md5_res = sudo(['openssl', 'md5'], stdin=content.aggr_stdout)
+    md5_hash = md5_res.aggr_stdout.split('(stdin)= ')[1]
+    return md5_hash
+
+
+def two_certs_identical(cert_a, cert_b):
+    return _get_md5_hash(cert_a) == _get_md5_hash(cert_b)
```

Comment:
I'm not clear why you're comparing these using an md5sum rather than just a straight comparison- and why you're doing so using the openssl md5 command rather than just md5sum?

Reply:
1. Based on the discussion here https://serverfault.com/questions/913144/how-can-i-compare-two-ssl-certificates, two certificates can have the same attributes, but they would seem different if they were generated at different times. This is the reason to use md5sum. But there might be a better way I don't know about.

2. I didn't know python has a md5sum command, so I just implemented the solution according to the third answer in the website above. I will change to use the md5sum function from hashlib package (looked for it now).

Reply:
I thought about it again. It makes no sense to use the hashing function. Will just compare the files.